tests/e2e/specs/modules/search-console/dashboard-date-range.test.js

```diff
@@ -36,7 +36,16 @@ import * as dashboardRequests from './fixtures/dashboard';
 import * as dashboardDetailsRequests from './fixtures/dashboard-details';
 import * as modulePageRequests from './fixtures/module-page';
 
+// TODO The dashboard and page dashboard tests still use legacy widgets
+// and require legacy fixtures via calls to /data. Once they are refactored, this
+// can be removed and the request interception can be brought in line with the other
+// Widget API tests (see below).
```

Comment:
The issue isn't that the tests are using legacy widgets but the actual pages are, right?

Let's update this to say so:

```suggestion
// TODO The dashboard and page dashboard still use legacy widgets
// and require legacy fixtures via calls to /data. Once they are refactored, this
// can be removed and the request interception can be brought in line with the other
// Widget API tests (see below).
```

Reply:
Ah yes! Good spot.